### PR TITLE
Fix/minor issues

### PR DESCRIPTION
**Pull request by nicolearagao**
Minor fixes to mainly address disparities between current implementation and UX sketches.
- refactor order of tables
- fix(source): re-order fields to match UX sketches
- fix(sources): avoid inconsistencies in sources
- fix(scans): use correct icon when no scans available
- fix(sources): correct no sources available text

### CODE REVIEW

File: src/pages/sources/components/AddSourceModal.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const AddSourceModal: React.FC<AddSourceModalProps> = ({ source, type, onClose,
       credentials: credentials.map(c => Number(c)),
       hosts: values['hosts'].split(','),
       name: values['name'],
-      port: !isNetwork ? '443' : values['port'] || '22',
+      port: !isNetwork ? values['port'] || '443' : values['port'] || '22',
```
Comment (gitdallas) [inline src/pages/sources/components/AddSourceModal.tsx:82]: so values can always override, even if not `isNetwork`?

if so, probably better to do like... `values['port'] || (isNetwork ? '22' : '443')`

**Review by gitdallas — APPROVED** [review]
looks good, one nit i commented on

File: src/pages/sources/components/AddSourceModal.tsx [hunk 2]
```diff
@@ -155,24 +162,32 @@ const AddSourceModal: React.FC<AddSourceModalProps> = ({ source, type, onClose,
...
+                    value={getValue('port')}
+                    placeholder="Optional"
+                    type="text"
+                    id="source-port"
+                    name="port"
+                    onChange={ev => {
+                      setValue('port', (ev.target as HTMLInputElement).value);
+                    }}
+                  />
+                  <HelperText>Default port is 443</HelperText>
```
Comment (ruda) [inline src/pages/sources/components/AddSourceModal.tsx:187]: For what source type this 443 (HTTPS) refers to? I mean, for network (SSH) the default port is 22.

Comment (nicolearagao) [reply src/pages/sources/components/AddSourceModal.tsx:187]: yes, network default port is specified somewhere else.

Comment (nicolearagao) [reply src/pages/sources/components/AddSourceModal.tsx:187]: values['port'] || (isNetwork ? '22' : '443')

File: src/pages/sources/components/AddSourceModal.tsx (hunk 1 above)
Comment (nicolearagao) [reply src/pages/sources/components/AddSourceModal.tsx]: fixed

**Pull request merged**